{{cookiecutter.slug}}/bootstrap.py

```diff
@@ -163,6 +167,98 @@ def make_create_db_command(psql_cmd):
         psql_cmd + ' -f ' + op.join('backend', 'create_db.sql'),
     )
 
+def merge_json(target, source):
+    for key, value in source.items():
+        if value == None:
+            del target[key]
+        elif key in target and isinstance(target[key], dict) and \
+            isinstance(source[key], dict):
+            merge_json(target[key], source[key])
+        else:
+            target[key] = value
+    return target
+
+def create_angular_localization_settings():
+    with open('frontend/angular.json', 'r') as file:
+        data = json.load(file)
+    project = '{{cookiecutter.slug}}'.replace('_', '-')
+    for lang in '{{cookiecutter.localizations}}'.split(','):
+        [code, lang_name] = lang.split(':')
+        production = merge_json({}, data['projects'][project]['architect']['build']['configurations']['production'])
+        production['outputPath'] = f'dist/{code}'
+        production['i18nFile'] = f'src/locale/messages.{code}.xlf'
+        production['i18nFormat'] = 'xlf'
+        production['i18nLocale'] = code
+        production['i18nMissingTranslation'] = 'error'
+        data['projects'][project]['architect']['build']['configurations'][f'production-{code}'] = production
+
+        serve = merge_json({}, data['projects'][project]['architect']['serve']['configurations']['production'])
+        serve['browserTarget'] += f'-{code}'
+        data['projects'][project]['architect']['serve']['configurations'][code] = serve
+
+    data['projects'][project]['architect']['build']['configurations']['production']['outputPath'] = 'dist/default'
+
+    with open('frontend/angular.json', 'w') as file:
+        json.dump(data, file, indent=2)
+
+def activate_frontend():
+    framework = '{{cookiecutter.frontend}}'
+    os.rename('package.{{cookiecutter.frontend}}.json', 'package.json')
+
+    if framework == 'backbone':
+        os.rename('frontend.backbone', 'frontend')
+    elif framework == 'angular':
+        project_name = '{{cookiecutter.slug}}'.replace('_', '-')
+        Command(
```

Comment:
I recommend creating these commands as global objects (i.e., in module scope) and just invoking them here. These objects have a `__print__` method and the invocation (`__call__`) returns a boolean which indicates whether the command was successful, so you can make one command conditional on the success of another and you can also display failed commands to the user so they can run them by themselves. See the `main` function for example usage.